### ISSUE TITLE
fix PHOTSYS='G'

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1132]
yet another Gaia-target case that wasn't caught by #1105, #1109, and #1114.  As reported by @akremin
```
desi_fit_stdstars --delta-color 0.1 \
--frames \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/frame-b6-00075147.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/frame-r6-00075147.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/frame-z6-00075147.fits \
--skymodels \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/sky-b6-00075147.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/sky-r6-00075147.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20210208/00075147/sky-z6-00075147.fits \
--fiberflats \
  /global/cfs/cdirs/desi/spectro/redux/daily/calibnight/20210208/fiberflatnight-b6-20210208.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/calibnight/20210208/fiberflatnight-r6-20210208.fits \
  /global/cfs/cdirs/desi/spectro/redux/daily/calibnight/20210208/fiberflatnight-z6-20210208.fits \
--starmodels \
  /global/cfs/cdirs/desi/spectro/templates/basis_templates/v3.2/stdstar_templates_v2.2.fits \
--outfile $SCRATCH/stdstars-6-00075147.fits
...
Traceback (most recent call last):
 File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/master/bin/desi_fit_stdstars", line 20, in <module>
  stdstars.main(args)
 File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/master/py/desispec/scripts/stdstars.py", line 449, in main
  model_filters[band+photsys] = load_legacy_survey_filter(band=band,photsys=photsys)
 File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/master/py/desispec/io/filters.py", line 50, in load_legacy_survey_filter
  raise ValueError("unknown photsys '{}', known ones are 'N' and 'S'".format(photsys))
ValueError: unknown photsys 'G', known ones are 'N' and 'S'
```

PHOTSYS='G' means that it is a known Gaia-only target (i.e. not matched to a DR9 target).  Some earlier tiles had this as PHOTSYS='', but we requested targeting to start flagging these as PHOTSYS='G' to differentiate them from locations that are genuinely not matched to anything (e.g. a transient).

@segasai could you submit a fix for this as part of supporting Gaia targets?  Thanks.

**Comment by segasai** [comment]
@sbailey I've submitted the PR  #1133

**Comment by sbailey** [comment]
Fixed by #1133.  Closing.
